stdlib/socket.pyi

```diff
@@ -203,6 +203,9 @@ if sys.platform != "win32" and sys.platform != "darwin":
         TCP_SYNCNT as TCP_SYNCNT,
         TCP_WINDOW_CLAMP as TCP_WINDOW_CLAMP,
     )
+
+    if sys.version_info >= (3, 6, 7):
```

Comment:
We only support Python 3.7+ in typeshed, so the `sys.version_info` guard here is unnecessary :)

Reply:
done!